examples/basic/variables.tf

```diff
@@ -1,5 +1,5 @@
 variable "powervs_zone" {
-  description = "IBM Cloud data center location where IBM PowerVS infrastructure will be created. Following locations are currently supported: syd04, syd05, eu-de-1, eu-de-2, tok04, osa21, sao01"
+  description = "IBM Cloud data center location where IBM PowerVS infrastructure will be created. Following locations are currently supported: syd04, syd05, eu-de-1, eu-de-2, tok04, osa21, sao01, lon04, lon06"
```

Comment:
how about adding a validation check to ensure only supported values are passed (https://developer.hashicorp.com/terraform/language/values/variables#custom-validation-rules)

Reply:
oh I see you have it in the main modules variables.tf, so maybe not needed here too

Reply:
Right